src/solana/exceptions.py

```diff
@@ -7,8 +7,10 @@ class SolanaExceptionBase(Exception):
 
     def __init__(self, exc: Exception, func: Callable[[Any], Any], *args: Any, **kwargs: Any) -> None:
         """Init."""
+    def __init__(self, exc: Exception, func: Callable[[Any], Any], *args: Any, **kwargs: Any) -> None:
         super().__init__()
         self.error_msg = self._build_error_message(exc, func, *args, **kwargs)
+        self.original_exception = exc
```

Comment:
I don't think we need to set this since `_build_error_message` is a static method